x-pack/plugins/maps/public/classes/sources/vector_source/vector_source.tsx

```diff
@@ -75,7 +75,6 @@ export interface IVectorSource extends ISource {
     defaultFields: Record<string, Record<string, string>>
   ): Promise<void>;
   deleteFeature(featureId: string): Promise<void>;
-  isFilterByMapBounds(): boolean;
```

Comment:
not relevant to this fix, but this method exists ISource and does not need to be duplicated in VectorSource